### PR TITLE
Update _index.en.md

### PR DESCRIPTION
**Pull request by yy2so**
Adding new committer avatar

---

Before committing this PR, I'm sure that I have checked the following options:
- [x] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [x] I have self-reviewed the commit code.
- [x] I have triggered maven check: `mvn clean install -B -T2C -DskipTests -Dmaven.javadoc.skip=true -e`.
- [x] I have made corresponding changes to the documentation.
- [x] I have added corresponding unit tests for my changes.

### CODE REVIEW

File: docs/community/content/team/_index.en.md [hunk 1]
```diff
@@ -422,14 +422,19 @@ chapter = true
                 <img src="https://avatars.githubusercontent.com/u/64654203?v=4" style="border-radius:50%; overflow:hidden">
             </a>
         </td>
+     <td>
+    </a href="https://github.com/yy2so" target="_blank">
+                <img src="https://avatars.githubusercontent.com/u/86716581?v=4" style="border-radius:50%; overflow:hidden">
```
Comment (TeslaCN) [inline docs/community/content/team/_index.en.md:427]:
```suggestion
        <td>
               <a href="https://github.com/yy2so" target="_blank">
                    <img src="https://avatars.githubusercontent.com/u/86716581?v=4" style="border-radius:50%; overflow:hidden">
               </a>
        </td>
```

**Review by TeslaCN — CHANGES_REQUESTED** [review]

File: docs/community/content/team/_index.en.md [hunk 2]
```diff
@@ -422,14 +422,19 @@ chapter = true
...
         <td>
             <b>Junjie Zhang</b>        
         </td>
         <td>
-            <b>Swastika Gupta</b>        
-        </td>
+            <b>Swastika Gupta</b>
+              <td>
+            <b>Yacine Si Tayeb</b>
+        </tr>
```
Comment (TeslaCN) [inline docs/community/content/team/_index.en.md:437]:
```suggestion
            <b>Swastika Gupta</b>
        </td>
        <td>
            <b>Yacine Si Tayeb</b>
        </td>
```

File: docs/community/content/team/_index.en.md [hunk 3]
```diff
@@ -422,13 +422,21 @@ chapter = true
                 <img src="https://avatars.githubusercontent.com/u/64654203?v=4" style="border-radius:50%; overflow:hidden">
             </a>
         </td>
+        <td>
+               <a href="https://github.com/yy2so" target="_blank">
+                    <img src="https://avatars.githubusercontent.com/u/86716581?v=4" style="border-radius:50%; overflow:hidden">
+               </a>
```
Comment (TeslaCN) [inline docs/community/content/team/_index.en.md:428]:
```suggestion
            <a href="https://github.com/yy2so" target="_blank">
                   <img src="https://avatars.githubusercontent.com/u/86716581?v=4" style="border-radius:50%; overflow:hidden">
            </a>
```

File: docs/community/content/team/_index.en.md [hunk 4]
```diff
@@ -422,13 +422,21 @@ chapter = true
                 <img src="https://avatars.githubusercontent.com/u/64654203?v=4" style="border-radius:50%; overflow:hidden">
             </a>
         </td>
+        <td>
+            <a href="https://github.com/yy2so" target="_blank">
+                   <img src="https://avatars.githubusercontent.com/u/86716581?v=4" style="border-radius:50%; overflow:hidden">
```
Comment (TeslaCN) [inline docs/community/content/team/_index.en.md:427]:
```suggestion
                <img src="https://avatars.githubusercontent.com/u/86716581?v=4" style="border-radius:50%; overflow:hidden">
```

**Comment by TeslaCN** [comment]
Hi @yy2so 
The _index.cn.md need to be updated too.

**Comment by yy2so** [comment]
> 

@TeslaCN Done, updated CN version too

**Review by TeslaCN — APPROVED** [review]

**Pull request merged**